src/diseaseProg.jl

```diff
@@ -561,4 +561,53 @@ function inpFunc_testSpecifications(
     testSpecifications[17:21, "FalsePositiveRate"] .= antibody_FPR_S_to_I4
 
     return testSpecifications
+end
+
+function trFunc_testCapacity(
+    realTime::Date, # time within simulation (day)
+    # PCR capacity - initial
+    testCapacity_pcr_phe_total::Float64 = 1e4,
+    testCapacity_pcr_phe_inflexday::Date = Date("2020-03-25", "yyyy-mm-dd"),
+    testCapacity_pcr_phe_inflexslope::Float64 = 5.0,
+
+    # PCR capacity - increased
+    testCapacity_pcr_country_total::Float64 = 1e5,
+    testCapacity_pcr_country_inflexday::Date = Date("2020-04-25", "yyyy-mm-dd"),
+    testCapacity_pcr_country_inflexslope::Int64 = 10,
```

Comment:
Better keep every attribute `Float64`, until `type` is fixed and known.

Reply:
Sure.. Will change that

Reply:
If I change it to float64 then the argument has to be 10.0 instead of 10. Is that fine and is that what you want?